generated/nidmm/nidmm_library_interface.h

```diff
@@ -0,0 +1,115 @@
+  virtual ViStatus GetCalDateAndTime(ViSession vi, ViInt32 calType, ViInt32* month, ViInt32* day, ViInt32* year, ViInt32* hour, ViInt32* minute) = 0;
+  virtual ViStatus GetCalUserDefinedInfo(ViSession vi, ViInt32 bufferSize, ViChar info[256]) = 0;
+  virtual ViStatus GetCalUserDefinedInfoMaxSize(ViSession vi, ViInt32* infoSize) = 0;
+  virtual ViStatus GetDevTemp(ViSession vi, ViString options, ViReal64* temperature) = 0;
+  virtual ViStatus GetError(ViSession vi, ViStatus* errorCode, ViInt32 bufferSize, ViChar description[]) = 0;
+  virtual ViStatus GetErrorMessage(ViSession vi, ViStatus errorCode, ViInt32 bufferSize, ViChar errorMessage[]) = 0;
+  virtual ViStatus GetExtCalRecommendedInterval(ViSession vi, ViInt32* months) = 0;
+  virtual ViStatus GetLastCalTemp(ViSession vi, ViInt32 calType, ViReal64* temperature) = 0;
+  virtual ViStatus GetMeasurementPeriod(ViSession vi, ViReal64* period) = 0;
+  virtual ViStatus GetSelfCalSupported(ViSession vi, ViBoolean* selfCalSupported) = 0;
+  virtual ViStatus ImportAttributeConfigurationBuffer(ViSession vi, ViInt32 size, ViInt8 configuration[]) = 0;
+  virtual ViStatus ImportAttributeConfigurationFile(ViSession vi, ViConstString filePath) = 0;
+  virtual ViStatus init(ViRsrc resourceName, ViBoolean idQuery, ViBoolean resetDevice, ViSession* vi) = 0;
+  virtual ViStatus InitWithOptions(ViString resourceName, ViBoolean idQuery, ViBoolean resetDevice, ViString optionString, ViSession* vi) = 0;
+  virtual ViStatus Initiate(ViSession vi) = 0;
+  virtual ViStatus InvalidateAllAttributes(ViSession vi) = 0;
+  virtual ViStatus IsOverRange(ViSession vi, ViReal64 measurementValue, ViBoolean* isOverRange) = 0;
+  virtual ViStatus IsUnderRange(ViSession vi, ViReal64 measurementValue, ViBoolean* isUnderRange) = 0;
+  virtual ViStatus LockSession(ViSession vi, ViBoolean* callerHasLock) = 0;
+  virtual ViStatus PerformOpenCableComp(ViSession vi, ViReal64* conductance, ViReal64* susceptance) = 0;
+  virtual ViStatus PerformShortCableComp(ViSession vi, ViReal64* resistance, ViReal64* reactance) = 0;
+  virtual ViStatus Read(ViSession vi, ViInt32 maximumTime, ViReal64* reading) = 0;
+  virtual ViStatus ReadMultiPoint(ViSession vi, ViInt32 maximumTime, ViInt32 arraySize, ViReal64 readingArray[], ViInt32* actualNumberOfPoints) = 0;
+  virtual ViStatus ReadStatus(ViSession vi, ViInt32* acquisitionBacklog, ViInt16* acquisitionStatus) = 0;
+  virtual ViStatus ReadWaveform(ViSession vi, ViInt32 maximumTime, ViInt32 arraySize, ViReal64 waveformArray[], ViInt32* actualNumberOfPoints) = 0;
+  virtual ViStatus reset(ViSession vi) = 0;
+  virtual ViStatus ResetInterchangeCheck(ViSession vi) = 0;
+  virtual ViStatus ResetWithDefaults(ViSession vi) = 0;
+  virtual ViStatus RestoreLastExtCalConstants(ViSession vi) = 0;
+  virtual ViStatus SelfCal(ViSession vi) = 0;
+  virtual ViStatus self_test(ViSession vi, ViInt16* selfTestResult, ViChar selfTestMessage[256]) = 0;
```

Comment:
Use the tag 'cname' : 'niDMM_self_test' tag for this in functions.py. Refer to scope/switch metadata files.
Same goes for other APIs that use snake case in the API name.

Reply:
made the change